web_src/js/components/RepoContributors.vue

```diff
@@ -150,6 +163,7 @@ export default {
           return false;
         });
 
+        user['max_contribution_type'] += 1;
```

Comment:
I think this line is unnecessary. Without this change:

<img width="1086" alt="image" src="https://github.com/sahinakkaya/gitea/assets/32161460/dce07d75-02f2-4a2d-ab00-6fce25302977">


With this change:

<img width="1086" alt="image" src="https://github.com/sahinakkaya/gitea/assets/32161460/cc6eccf0-049b-4a7a-bcc8-45742072258e">


I will remove that line.

Reply:
In some places, it makes sense.
The data you looked is [0, 0, 1, 0, 0]
But if the data is [0, 0, 1, 1, 0, 0],  the head of this wave will be missed.

Before:
![image](https://github.com/sahinakkaya/gitea/assets/18380374/f7a7f890-e475-49bc-972d-389a1e829662)
After:
![image](https://github.com/sahinakkaya/gitea/assets/18380374/68cc6b0f-57c1-4c9a-a9eb-e691d5b873b5)


ps: or we can `+0.5`? or calculate it by `max_contribution_type`? For example, `max_contribution_type * k`, k can be 1.1 or 1.2 or other numbers.

Reply:
> The data you looked is [0, 0, 1, 0, 0]

But it says 2 commits at the top. How? @yp05327

Reply:
Sorry, it is [0, 0, 2, 0, 0].

Reply:
OK, I got it now. Can you suggest this change including a comment with a link to the problem? Something like: "this line is required because <link to the problem>"

I will then accept your change.

Reply:
I have already suggested in https://github.com/go-gitea/gitea/pull/27882/files#r1393549185.
Emmm, I will mention you there.
Or should I make a new comment?

Reply:
I wrote comment. Can you check it?